code/modules/customitems/item_defines.dm

```diff
@@ -748,6 +748,18 @@
 	desc = "A worn leather jacket. Some burn holes have been patched."
 	body_parts_covered = UPPER_TORSO|LOWER_TORSO|LEGS|ARMS
 	icon_state = "dusty_jacket"
+	
+/obj/item/clothing/suit/fluff/supplymaster_jacket //Denthamos: Henry Grandpa Gadow
+	name = "faded NT Supply Master's Coat"
+	desc = "A faded leather overcoat bearing a worn out badge from the NAS Crescent on the shoulder, and a designation tag of Supply master on the front.  A tarnished gold nameplate says H.Gadow on it."
+	body_parts_covered = UPPER_TORSO|LOWER_TORSO|LEGS|ARMS
+	icon_state = "supplymaster_jacket_open"
+	item_state = "supplymaster_jacket_open"
+	ignore_suitadjust = 0
+	suit_adjusted = 1
+	allowed = list(/obj/item/weapon/door_remote/quartermaster, /obj/item/device/megaphone, /obj/item/weapon/stamp/granted, /obj/item/weapon/stamp/denied, /obj/item/weapon/tank/oxygen/yellow, /obj/item/weapon/tank/emergency_oxygen)
```

Comment:
Is this in line with other suits?

Reply:
Aye, I straight copied it from the hazard vest code

Reply:
Just check that if the lab coat

Reply:
No it's not.
https://github.com/ParadiseSS13/Paradise/blob/16f0872ef0ea591c2e578bdb6ce3dfb60f7ea4ed/code/modules/clothing/suits/miscellaneous.dm

This seems to have the added ability of holding the QM's door remote, a megaphone and both granted and denied stamps

Reply:
I wasn’t aware you meant what the coat could hold, that was done by request of Denthamos. I don’t believe a pocket that holds a megaphone is too gameplay altering.

Reply:
It's a difference from the standard item that's not mentioned anywhere.
Fluff items are supposed to be *only* fluff, what you don't believe to be *too gameplay altering* doesn't matter, it's just a slippery slope.

Keep items purely fluff.

Reply:
You should probably be talking to denthamos about this btw

Reply:
Well, I have to say the fact it can hold one of 3 small items or a normal sized item that is unique from other jackets isn't that game altering.

Reply:
Eh if its too gameplay altering I do not really care about the additional items.  Only thing I would say would be the 02 tank should be fine considering the current fur coat item and winter coats can hold an 02 tank in exosuit slot so at that point is it standardization or are those items not meant to be able to hold such there?